systemtest/apmservertest/config.go

```diff
@@ -436,6 +436,12 @@ func DefaultConfig() Config {
 	}
 }
 
+// DefaultKibanaPort returns the Kibana port, configured using
+// KIBANA_PORT, or otherwise returning the default of 5601.
+func DefaultKibanaPort() string {
```

Comment:
this doesn't necessarily return the default port, should be called `KibanaPort()`?

Reply:
Good call, updated.